sdk/cosmos/azure-cosmos-kafka-connect/src/docker/resources/sink.example.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "cosmos-kafka-connectv2",
```

Comment:
nit: maybe naming it `cosmosdb-sink-connector-v2` would be better for consistency

Reply:
updated